include/osquery/status.h

```diff
@@ -54,13 +56,19 @@ class Status {
    */
   Status(int c, std::string m) : code_(c), message_(std::move(m)) {}
 
+  Status(std::shared_ptr<Error> error)
+      : code_(error->getErrorCode()),
+        message_(error->getFullMessageRecursive()) {}
+
  public:
   /**
    * @brief A getter for the status code property
    *
    * @return an integer representing the status code of the operation.
    */
-  int getCode() const { return code_; }
+  int getCode() const {
```

Comment:
You can declare this method (and `ok()`) as `noexcept`

Reply:
it's clan format modification, no logic changes here